tests/lockfile-compat.rs

```diff
@@ -52,18 +52,25 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
             foo = "0.1.0"
         "#)
         .file("src/lib.rs", "")
-        .file("Cargo.lock", lockfile);
+        .file("Cargo.lock", old_lockfile);
     p.build();
 
-    assert_that(p.cargo("build"),
-                execs().with_status(0));
+    let cargo_commands = vec![
+        "build",
+        "update"
+    ];
 
-    let lock = p.read_lockfile();
-    for (l, r) in expected_lock_file.lines().zip(lock.lines()) {
-        assert!(lines_match(l, r), "Lines differ:\n{}\n\n{}", l, r);
-    }
+    for cargo_command in cargo_commands {
```

Comment:
Hm, I think this won't work as expected, because the second time around the lockfile will already be updated.

Reply:
Ah! of course. I'll probably create a helper method and loop through the vector.